packages/blade/src/components/Badges/_decisions/decisions.md

```diff
@@ -0,0 +1,77 @@
+# Badge <!-- omit in toc -->
+Badges are used to show small amount of color coded metadata, which are ideal for getting users attention.
+
+<img src="./badge-thumbnail.png" width="380" />
+
+## Design
+- [Figma - Badge](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8110%3A417)
+
+
+## API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| children | `string` | `undefined` | The label to be rendered within the badge. | ✅ |
+| variant | `positive`,`negative`, `notice`, `information`, `neutral` | `neutral` | The variant of the badge to be rendered. |  |
+| contrast | `low`, `high` | `low` | The contrast of the badge to be rendered. |  |
+| size | `medium`, `small` | `medium` | The size of the badge to be rendered. |  |
```

Comment:
What is the difference b/w the medium and small sizes? Shouldn't the badge wrap its content and have a `max-width`?

Reply:
Different heights. Small is of 20px and Medium is 24px